src/writer.rs

```diff
@@ -400,6 +400,20 @@ impl<'b> UiContext<'b> {
                                 self.need_redraw = true;
                                 return Ok(false);
                             }
+                            KeyCode::Backspace => {
+                                let s_len = s.chars().count();
+                                if s_len == 0 {
```

Comment:
You don't need to use count() check pop() return value whether Some or None

Reply:
> You don't need to use count() check pop() return value whether Some or None

Haha, good point. Fixed.